src/game/server/entities/character.cpp

```diff
@@ -857,7 +857,7 @@ void CCharacter::Snap(int SnappingClient)
 
 	if(pCharacter->m_Emote == EMOTE_NORMAL)
 	{
-		if(250 - ((Server()->Tick() - m_LastAction)%(250)) < 5)
+		if(5 * SERVER_TICK_SPEED - ((Server()->Tick() - m_LastAction) % (5 * SERVER_TICK_SPEED)) < 5)
```

Comment:
could make sense to also refactor the magic 5 ticks as emote-tick

Reply:
I think the last 5 doesn't mean the same as the other 5. As far as I understand it, this code shows the blink emote for the last 5 ticks in every 5 seconds since a player's last input. So those 5s could also be different numbers. Probably overkill to add a constant, at least right now.